### DISCUSSION TITLE
How to go back to my last progress

### DISCUSSION THREAD

**Discussion by randy-chen2022** [xournalpp/xournalpp#4383] [Q&A]
I can't finish editing a document every time, and the next time I open it, the document starts from the beginning. So, is there a function that allows the document to automatically remember the page number when the document is closed, or I haven't found it?

**Comment by bhennion** [comment]
I think what you're looking for is in Preferences -> Load/Save -> Autoloading -> Enable autoloading of most recent file on application startup

**Reply by randy-chen2022** [reply]
Thank you for answering. I want the document, whereas, to automatically navigate to the last edited page instead of automatically opening the file. Sadly, the settings you provided did not live up to my expectations, it do open the last file but still start from the first page.

**Reply by bhennion** [reply]
There is not way to do that in the current version
